android/app/src/main/kotlin/org/getlantern/lantern/util/PaymentsUtil.kt

```diff
@@ -245,17 +245,18 @@ class PaymentsUtil(private val activity: Activity) {
                         * make user pro temporary next user open app it will check the purchase state and call api accordingly
                         * */
                         session.linkDevice()
+                        session.setIsProUser(true)
```

Comment:
Added changes here,
@atavism makes user pro regress of userData API call, If payment has been processed then it pro temporary, Next user starts the app it will fetch data from server.